queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -3,8 +3,11 @@
 
 namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.Extensions
 {
+    // Static class containing extension methods for configuring receipt commands for the DEFAULT queue.
     public static class ServiceCollectionExtensions
     {
+        // Method to configure and register various receipt command types with the dependency injection container.
+        // This method adds different types of receipt commands making them available for the rest of the application.
```

Comment:
I be super clear here and add something like "Receipt commands need to be registered here".